resources/icons/vscode/data/extension_to_icon_name_map.rs

```diff
@@ -0,0 +1,12 @@
+// Tips on editing this file in vi, use C-n and C-y in edit mode
+[
+	
```

Comment:
I got tired and didn't add too many extensions. This can probably be crowdsourced :)

Reply:
No problem. I'll work on the integration my side and if it's OK we'll do the boring parts later.

Reply:
I'm going to keep adding a few as I use this more ...